_data/locales/ja.yml

```diff
@@ -18,10 +18,10 @@ ja:
       install: インストール
       paste: これをmacOSのターミナルまたはLinuxのシェルプロンプトに貼り付けて下さい。
       what: スクリプトは何をするのか説明してから、それを開始する前に一度中断します。その他の<a href="https://docs.brew.sh/Installation">インストールオプション</a>も確認してください。
-      pkginstaller: If you're on macOS, try our new <code>.pkg</code> installer.
-      githubrelease: Download it from <a href="https://github.com/Homebrew/brew/releases/latest">Homebrew's latest GitHub release</a>.
+      pkginstaller: macOSをお使いの場合は新しい<code>.pkg</code>インストーラーをお試し下さい。
+      githubrelease: <a href="https://github.com/Homebrew/brew/releases/latest">Homebrewの最新のGitHubリリース</a>からダウンロードして下さい。
     doc:
-      man: <code>man brew</code> documentation
+      man: <code>brew</code>コマンドのドキュメント
```

Comment:
This section was changed to follow [English version](https://github.com/homebrew/brew.sh/blob/68828eaf6da57357080435a4c23abb0ee172e385/_data/locales/en.yml#L24).